### PR TITLE
Fix Kafka source yaml

### PR DESCRIPTION
**Pull request by aliok**
Doing the YAML changes of https://github.com/openshift/knative-eventing-contrib/pull/612 on master

### CODE REVIEW

**Review by matzew — APPROVED** [review]
/lgtm
/approve

File: knative-operator/deploy/resources/knativekafka/kafkasource-v0.17.1.yaml [hunk 1]
```diff
@@ -196,6 +201,12 @@ spec:
   scope: Namespaced
   subresources:
     status: {}
+  conversion:
+    strategy: Webhook
+    webhookClientConfig:
+      service:
+        name: kafka-source-webhook
+        namespace: knative-sources
```
Comment (matzew) [inline knative-operator/deploy/resources/knativekafka/kafkasource-v0.17.1.yaml:209]: The `knative-eventing` Namespace should be used

**Comment by aliok** [comment]
thanks @matzew for catching that!

need a new lgtm and unhold

**Review by matzew — APPROVED** [review]
/lgtm
/approve
/unhold

**Pull request merged**